control-plane/config/eventing-kafka-broker/100-broker/100-kafka-broker-configmap.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   name: kafka-broker-config
   namespace: knative-eventing
   labels:
-    kafka.eventing.knative.dev/release: devel
+    app.kubernetes.io/version: devel
```

Comment:
I think in eventing (and serving) we kept the old labels, for a release, and afterwards removed those.

would that route make sense here too?